db.js

```diff
@@ -139,12 +145,29 @@ const OrderDetail = conn.define('orderDetail', {
   }
 });
 
+const Cart = conn.define('cart', {
```

Comment:
I think this model will get you in trouble, I recommend using a relationship more like:

Product -> LineItem -> Cart

Where 
```typescript
interface Product {
  name: string;
  // ...
}

interface LineItem {
  productId: string;
  cartId: string;
  quantity: number;
}

interface Cart {
  lineItems: LineItem[];
}
```